ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -884,6 +899,23 @@ literals:
 - valve_status:
   - OPEN
   - CLOSED
+- zone_control_cooling_command:
+  - COOLING
+  - OFF
+- low_battery_alarm:
+  - ACTIVE
+  - INACTIVE
+- load_bypass_mode:
```

Comment:
is the device ever `ON`? seems like some states are missing; otherwise this should be load_bypass_status and it should be `NORMAL`/`BYPASS` (or similar)

Reply:
ah, there is a separate status for the device sir (run_status). this load_bypass_mode is just on/off

Reply:
noted sir.